### PR TITLE
Adding RazorrBoat

### PR DESCRIPTION
**Pull request by Squazy**
This pull request was made by the Gatsby Branch Editor.

### CODE REVIEW

File: data/en/bots/435454917616861197.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+avatar: >-
+  https://images.discordapp.net/avatars/435454917616861197/8a350b3ded637ad480cfce7f49773afd.png
+cover: null
+custom_path: null
+description: >-
+  You can create your server with this bot in one command. All you have to do is
+  make sure the bot has all permissions and you do also then do r!setup and
+  follow the steps.
```
Comment (Andre601) [inline data/en/bots/435454917616861197.md]: You have to vote for the bot, to actually use rge r!setup command.... Not a good thing dude.

Comment (lepon01) [reply data/en/bots/435454917616861197.md]: Is this .org voting?

File: data/en/bots/435454917616861197.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+github:
+  owner: squazy
```
Comment (Andre601) [inline data/en/bots/435454917616861197.md]: imo should you not add yourself for GitHub, if you don't have a repository to share.

**Comment by Stylite** [comment]
It is .org voting

**Comment by lepon01** [comment]
Requires voting on https:/\/discordbots.org/ voting

**Comment by lepon01** [comment]
voting on the vote for voting

**Comment by Andre601** [comment]
One thing I recommend is, that you change, how channels are created.
You can create channels directly through categories, eliminating the need, to sort them after creation ;)

**Comment by Andre601** [comment]
Also why do you have to provide a guild-name?
In 90% of cases, people already have their guildname set, so setting it through the command is a not really needed step.
You could however add, that people can provide an image (through a link or an attachment) and the bot uses it as a guild-icon...

**Pull request closed without merging**